site/operational-guide/interactive.md

```diff
@@ -2,55 +2,56 @@
 title: Interactive Deployment
 menu_order: 20
 ---
+Since launching Weave commands in interactive mode is not amenable to automation and
+configuration management, it is recommended that launching Weave in interactive 
+mode be used for exploration and evaluation only. However, if Weave is launched in 
+this mode, and as long as Docker is configured to start on boot,the resulting 
+network will survive host reboots without the use of a systemd.
```

Comment:
As an introduction, this should first state what this mode *is* for, before qualifying that with what it *isn't*. The original did that better though is by no means perfect.